README.md

```diff
@@ -133,4 +140,4 @@ WebHostingHub Glyphs
 - License: http://creativecommons.org/licenses/by/3.0/
 
 Splashy Icons (free as in free) 
-- Web page: http://splashyfish.com/icons/
```

Comment:
Done! Now PR 2011 contains original README.txt. Please check and let us know if it is fne :)